theme/src/components/SearchInput.tsx

```diff
@@ -21,7 +22,25 @@ export function SearchInput({ large, ...props }: Props) {
   const router = useRouter();
 
   // Local state for query. This is used to store the current entered query string.
-  const [localQuery, setLocalQuery] = useState('');
+  const [localQuery, setLocalQuery] = useState(() => {
+    const params = new URL(router.asPath, 'http://tmp.com').searchParams;
```

Comment:
why the temp/filler website url here?

Reply:
@kne42  I also had the same question in the last PR. See https://github.com/napari/napari.github.io/pull/156#discussion_r706519448

Reply:
I think for this, I'll create a utility function for creating temporary URLs with documentation on why it's necessary. The TLDR is that `new URL(...)` requires a base URL if the first argument is only a path like `/search`. Since `router.asPath` returns only a path, we need to supply a base so that `new URL(...)` doesn't [throw an error](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#exceptions). We only care about the search params part of the URL, so the `http://tmp.com` part is arbitrary.

Reply:
Added a util function `createUrl` 🍜